poetry.lock

```diff
@@ -343,58 +385,78 @@ mkdocs = ">=1.2.3"
 
 [[package]]
 name = "mkdocs-awesome-pages-plugin"
-version = "2.9.2"
+version = "2.10.1"
 description = "An MkDocs plugin that simplifies configuring page titles and their order"
 optional = false
-python-versions = ">=3.7"
+python-versions = ">=3.8.1"
+groups = ["main"]
 files = [
-    {file = "mkdocs_awesome_pages_plugin-2.9.2-py3-none-any.whl", hash = "sha256:9c795587695bd1ee85a8b7e43293005418df5a8b9ef296a3e628be427b693b4d"},
-    {file = "mkdocs_awesome_pages_plugin-2.9.2.tar.gz", hash = "sha256:c3f7d366ecfe99b64524c49a84d8e13c576c19a918ea2e6f59bb486a259313af"},
+    {file = "mkdocs_awesome_pages_plugin-2.10.1-py3-none-any.whl", hash = "sha256:c6939dbea37383fc3cf8c0a4e892144ec3d2f8a585e16fdc966b34e7c97042a7"},
+    {file = "mkdocs_awesome_pages_plugin-2.10.1.tar.gz", hash = "sha256:cda2cb88c937ada81a4785225f20ef77ce532762f4500120b67a1433c1cdbb2f"},
 ]
 
 [package.dependencies]
 mkdocs = ">=1"
 natsort = ">=8.1.0"
 wcmatch = ">=7"
 
+[[package]]
+name = "mkdocs-get-deps"
+version = "0.2.0"
+description = "MkDocs extension that lists all dependencies according to a mkdocs.yml file"
+optional = false
+python-versions = ">=3.8"
+groups = ["main"]
+files = [
+    {file = "mkdocs_get_deps-0.2.0-py3-none-any.whl", hash = "sha256:2bf11d0b133e77a0dd036abeeb06dec8775e46efa526dc70667d8863eefc6134"},
+    {file = "mkdocs_get_deps-0.2.0.tar.gz", hash = "sha256:162b3d129c7fad9b19abfdcb9c1458a651628e4b1dea628ac68790fb3061c60c"},
+]
+
+[package.dependencies]
+mergedeep = ">=1.3.4"
+platformdirs = ">=2.2.0"
+pyyaml = ">=5.1"
+
 [[package]]
 name = "mkdocs-material"
-version = "9.4.2"
```

Comment:
I don't know if the following should be done in this issue or in a new issue.

The `deploy-docs` GHA does a separate install of mkdocs and plugin packages. When the version we use changes, we should update it in the GHA file `.github/workflows/deploy-docs.yml` as well. In this case, we should update the `mkdocs-material` version in the GHA file to `9.6.8`.

There may be a way to have the GHA just read from `requirements.txt` like just passing the path to it in, but I didn't spend any time on it. I just copied the GHA example from the `mkdocs-material` docs.

I think the code change is not much, but there's testing and also documentation updates.

Reply:
Sure! Makes sense to do it in the same PR. I'll take a look.

Reply:
Ok, updated `mkdocks-material` in `deploy-docs.yml`.